build/root/.bazelrc

```diff
@@ -11,3 +11,6 @@ build --sandbox_tmpfs_path=/tmp
 # Ensure that Bazel never runs as root, which can cause unit tests to fail.
 # This flag requires Bazel 0.5.0+
 build --sandbox_fake_username
+
+# Run all tests with race detector
+test --features=race
```

Comment:
another possibility is making this
```
test:race --features=race
```
and then it'd only be enabled if you `bazel test --config=race`
(though I'm not sure how that'd interact with the Makefile)